apps/AppWithWearable/lib/pages/settings/page.dart

```diff
@@ -102,7 +102,7 @@ class _SettingsPageState extends State<SettingsPage> {
                   ...getPreferencesWidgets(
                     onOptInAnalytics: () {
                       setState(() {
-                        optInAnalytics = false;
+                        optInAnalytics = !SharedPreferencesUtil().optInAnalytics;
                         SharedPreferencesUtil().optInAnalytics = !SharedPreferencesUtil().optInAnalytics;
```

Comment:
[nitpick] Could've used the value of the variable above. (Not that the compiler won't be smart enough and optimize this...)